vic/drivers/classic/src/set_output_defaults.c

```diff
@@ -41,300 +41,266 @@ set_output_defaults(out_data_struct *out_data)
-        }
-        if (options.PRT_SNOW_BAND) {
-            options.Noutfiles++;
-        }
-        if (options.LAKES) {
-            options.Noutfiles++;
-        }
-        out_data_files = (out_data_file_struct *) calloc(options.Noutfiles,
-                                                         sizeof(
-                                                             out_data_file_struct));
-        filenum = 0;
-        strcpy(out_data_files[filenum].prefix, "fluxes");
-        if (options.FULL_ENERGY || options.FROZEN_SOIL) {
-            out_data_files[filenum].nvars = 26;
-        }
-        else {
-            out_data_files[filenum].nvars = 20;
-        }
+        out_data_files[filenum].nvars = 20;
+    }
+    filenum++;
+    strcpy(out_data_files[filenum].prefix, "snow");
+    if (options.FULL_ENERGY || options.FROZEN_SOIL) {
+        out_data_files[filenum].nvars = 14;
+    }
+    else {
+        out_data_files[filenum].nvars = 4;
+    }
+    if (options.BLOWING) {
+        out_data_files[filenum].nvars += 3;
+    }
+    if (options.FROZEN_SOIL) {
         filenum++;
-        strcpy(out_data_files[filenum].prefix, "snow");
-        if (options.FULL_ENERGY || options.FROZEN_SOIL) {
-            out_data_files[filenum].nvars = 14;
+        strcpy(out_data_files[filenum].prefix, "fdepth");
+        out_data_files[filenum].nvars = 4;
+    }
+    if (options.PRT_SNOW_BAND) {
+        filenum++;
+        strcpy(out_data_files[filenum].prefix, "snowband");
+        if (options.FULL_ENERGY) {
+            out_data_files[filenum].nvars = 13;
         }
         else {
-            out_data_files[filenum].nvars = 4;
-        }
-        if (options.BLOWING) {
-            out_data_files[filenum].nvars += 3;
-        }
-        if (options.FROZEN_SOIL) {
-            filenum++;
-            strcpy(out_data_files[filenum].prefix, "fdepth");
-            out_data_files[filenum].nvars = 4;
-        }
-        if (options.PRT_SNOW_BAND) {
-            filenum++;
-            strcpy(out_data_files[filenum].prefix, "snowband");
-            if (options.FULL_ENERGY) {
-                out_data_files[filenum].nvars = 13;
-            }
-            else {
-                out_data_files[filenum].nvars = 9;
-            }
-        }
-        if (options.LAKES) {
-            filenum++;
-            strcpy(out_data_files[filenum].prefix, "lake");
-            out_data_files[filenum].nvars = 8;
-        }
-        for (filenum = 0; filenum < options.Noutfiles; filenum++) {
-            out_data_files[filenum].varid = (unsigned int *) calloc(
-                out_data_files[filenum].nvars, sizeof(unsigned int));
+            out_data_files[filenum].nvars = 9;
         }
+    }
+    if (options.LAKES) {
+        filenum++;
+        strcpy(out_data_files[filenum].prefix, "lake");
+        out_data_files[filenum].nvars = 8;
+    }
+    for (filenum = 0; filenum < options.Noutfiles; filenum++) {
+        out_data_files[filenum].varid = (unsigned int *) calloc(
```

Comment:
I thought we had agreed to remove this style of `calloc` - or is that still pending?